plugins/action/synchronize.py

```diff
@@ -342,7 +342,9 @@ def run(self, tmp=None, task_vars=None):
 
         # Determine if we need a user@ and a password
         user = None
-        password = task_vars.get('ansible_ssh_pass', None) or task_vars.get('ansible_password', None)
+        password = (task_vars.get('ansible_ssh_password', None)
+                    or task_vars.get('ansible_ssh_pass', None)
+                    or task_vars.get('ansible_password', None))
```

Comment:
This order is the password priority order. In this case, this logic is a first match, so how about this order based on the documents as follows: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/connection/ssh.py#L61-L64

```
          vars:
              - name: ansible_password
              - name: ansible_ssh_pass
              - name: ansible_ssh_password
```